pkg/console/operator/sync_v400.go

```diff
@@ -358,6 +364,41 @@ func (co *consoleOperator) SyncConfigMap(
 	return cm, cmChanged, "ConsoleConfigBuilder", cmErr
 }
 
+func (co *consoleOperator) SyncClusterCAConfigMaps(ctx context.Context, operatorConfig *operatorv1.Console, recorder events.Recorder) ([]*corev1.ConfigMap, bool, string, error) {
+	managedClusters, err := co.managedClusterClient.ManagedClusters().List(ctx, metav1.ListOptions{LabelSelector: fmt.Sprintf("local-cluster!=true")})
+	if err != nil || len(managedClusters.Items) == 0 {
+		return nil, false, "", nil
+	}
+
+	configMaps := []*corev1.ConfigMap{}
+	errorReason := ""
+	modified := false
+	for _, managedCluster := range managedClusters.Items {
+		clusterName := managedCluster.GetName()
+		clientConfigs := managedCluster.Spec.ManagedClusterClientConfigs
+		if clientConfigs == nil || len(clientConfigs) == 0 {
+			klog.V(4).Infoln(fmt.Sprintf("Skippping API server CA ConfigMap for managed cluster %v, no client config found", clusterName))
+			continue
+		}
+		caBundle := clientConfigs[0].CABundle
+		if caBundle == nil {
+			klog.V(4).Infoln(fmt.Sprintf("Skippping API server CA ConfigMap for managed cluster %v, ca bundle not found", clusterName))
+			continue
+		}
+		required := configmapsub.DefaultClusterCAConfigMap(clusterName, caBundle, operatorConfig)
+		configMap, configMapModified, configMapApplyError := resourceapply.ApplyConfigMap(co.configMapClient, recorder, required)
+		modified = configMapModified || modified
+		if configMapApplyError != nil {
+			klog.V(4).Infoln(fmt.Sprintf("Skippping API server CA ConfigMap for managed cluster %v, CA ConfigMap not applied", clusterName))
+			err = configMapApplyError
+			errorReason = "FailedApply"
+			continue
```

Comment:
Shouldn't we break here? since we are setting an error ?

Reply:
I think maybe `err` should become a slice `errors []error`, then we can push to it and check that it has length > 0 after the loop. I can't break here because that would cause every subsequent managed cluster not to get configured.